packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-handlers/handler-tools.ts

```diff
@@ -0,0 +1,8 @@
+import { SessionIdNotFoundError } from "../errors/handler-errors";
```

Comment:
maybe let's call this file handler-utils?

Reply:
ok.